### ISSUE TITLE
网络的预训练模型

### ISSUE THREAD

**Issue by 10qianghua** [WangWenhao0716/Adapted-Center-and-Scale-Prediction#15]
作者您好：我在用您的resnetv2sn50这个网络跑代码的时候，出现了预训练模型的报错问题，报错为，找不到./models/ResNet101v2+SN(8,32).pth的路径，应该是路径问题，想知道您是怎么解决的。如何预训练的这个网络。或者有没有训练好的网络模型

**Comment by WangWenhao0716** [comment]
Please double-check the Preparation part in the READ.ME

The backbone of our ACSP is modified ResNet-101, i.e. replacing all BN layers with SN layers. You can download the pre-trained model from here. It is provided by the author of Switchable Normalization. The weights will be stored in ./models/.

**Comment by 10qianghua** [comment]
谢谢，我想用您这个方法的resnet50网络做实验，但是您这个上面只有101的预训练模型，您这边有没有已经预训练好的resnet50的网络模型，如果有的话，可否分享一下，急用，万分感谢

**Comment by 10qianghua** [comment]
'./net/pretrained/ResNet50v2+SN(8,32)-77.57.pth'

**Comment by WangWenhao0716** [comment]
https://pan.baidu.com/s/1XdN4tQcvTTIZjyNrnQpQXw

**Comment by 10qianghua** [comment]
万分感谢！
